_config.yml

```diff
@@ -68,3 +74,52 @@ contentful:
         other:
           layout: location
           draft: false
+    series:
+      id: series
+      body: description
+      associations:
+        videos:
+          - messages
+      frontmatter:
+        entry_mappings:
+          title: title
+          slug: slug
+          image: image/url
+          bg_image: background_image/url
+          start_date: starts_at
+          end_date: ends_at
+          youtube_url: youtube_url
+          apple_podcasts_url: apple_podcasts_url
+          google_play_url: google_play_url
+          year: "{{ starts_at | date: '%Y' }}"
+          videos: videos/id
+        other:
+          layout: series
+          draft: false
+    messages:
```

Comment:
Now that we're pulling in series and messages into this project, we have a new urgency to limit the volume of content coming back from Contentful. For example, in production today we have 1616 message records which will dramatically lengthen our build times. 

I'd suggest that we fix this issue in jekyll-contentful (via predecessor or blocking user-story) by providing some sort of mechanism to limit the number of records that are pulled back. We really only need the most recent series/messages content right?

Reply:
I think today it would just be the current series and last message.